.example.env

```diff
@@ -3,17 +3,20 @@
 S3API_SERVICE_PORT='5005'
 
 ## Auth:
-
-KEYCLOAK_ID='ID-string'
-KEYCLOAK_SECRET='secret-string'
-KEYCLOAK_ISSUER='issuer-string'
 KEYCLOAK_PUBLIC_KEYS_URL='public-keys-url-string'
 
 #for getting presigned URL
 URL_EXP_DAYS=7
 #temp subprefix in bucket that will be written to when arhicving and zippping
 TEMP_PREFIX='downloads-temp'
 
+# Auth
+AUTH_LEVEL='1'                                      # Options: [0, 1] corresponds to [role based auth, FGAC] (Optional).
+AUTH_ADMIN_ROLE='s3_admin'
+AUTH_SERVICE_ROLE='service_account'
+AUTH_SUPER_WRITER_ROLE='s3_super_writer'
```

Comment:
This is in a configuration file, so different deployments can use different `AUTH_SUPER_WRITER_ROLE` value without breaking the app. 

Honestly, all the role name values should be in env/config file not hardcoded like how they are right now, I didn't change the existing hardcoded values, since that is beyond the scope of this branch.

Reply:
I changed the example value to s3_writer.

Reply:
Thank you for changing it -- that makes it consistent with the hard coded values, but yes, I agree the hard coded values should be in the env down the road